sdk/core/azure-core/src/main/java/com/azure/core/util/tracing/Tracer.java

```diff
@@ -62,6 +62,11 @@
      */
     String SPAN_BUILDER_KEY = "builder";
 
+    /**
+     * Key for {@link Context} which indicates the the time of the last enqueued message in the partition's stream.
+     */
+    String MESSAGE_ENQUEUED_TIME = "x-opt-enqueued-time";
```

Comment:
Same constant is declared in OpenTelemetryTracer too. Can this defined in just one place?